code/script.js

```diff
@@ -1,12 +1,40 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const nameForm = document.getElementById('name-form')
+const nameInput = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper')
 
-// Global variables, if you need any, declared here
+let confirmation =''
 
-// Functions declared here
+const pizzaChoice = `
+  <select class="food-select" id='select'>
+    <option value='' selected disabled>Click here to choose pizza</option>
+    <option value='Margarita'>Margarita</option>
+    <option value='Hawaii'>Hawaii</option>
+    <option value='Calzone'>Calzone</option>
+  </select>
+`
```

Comment:
Nice idea to make external structures like this. Try to look a little bit closer if you can make only one and populate it with the use of `${}`. this skill will be very helpful in future projects 😸 You can always ask for more tips 😸